### PR TITLE
[Merged by Bors] - chore: move more dep to workspace

### PR DESCRIPTION
**Pull request by sehz**

### CODE REVIEW

File: release-tools/check-crate-version/toml-diff/Cargo.toml
Comment (morenol) [inline release-tools/check-crate-version/toml-diff/Cargo.toml:1]: but we need to add this crate to `Cargo.toml` members

Comment (sehz) [reply release-tools/check-crate-version/toml-diff/Cargo.toml:1]: it is using older version of toml: https://github.com/infinyon/fluvio/pull/3090.   can @tjtelan or @morenol update that PR?

Comment (morenol) [reply release-tools/check-crate-version/toml-diff/Cargo.toml:1]: I can do it

**Comment by sehz** [comment]
bors retry

**Comment by sehz** [comment]
bors retry

**Comment by sehz** [comment]
bors retry

**Comment by sehz** [comment]
bors retry

**Pull request closed without merging**